subprojects/test-runner/device-provider/impl/src/main/kotlin/com/avito/android/runner/devices/internal/kubernetes/KubernetesReservationClient.kt

```diff
@@ -48,6 +49,7 @@ internal class KubernetesReservationClient(
             if (state !is State.Idling) {
                 throw IllegalStateException("Unable claim reservation. State is already started")
             }
+            listener.onClaim(reservations)
```

Comment:
you used full name 'kubernetesReservationListener' in `subprojects/test-runner/device-provider/impl/src/main/kotlin/com/avito/android/runner/devices/internal/kubernetes/KubernetesReservationClaimer.kt`

but only `listener` here

because it's single listener here? i think it's better to use full name, easier to read

Reply:
Agree